src/docs/rialto-servermanager-api.json

```diff
@@ -0,0 +1,314 @@
+          }
+        },
+        "required": [
+          "id"
+        ]
+    },
+    "stopRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "deactivateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "getStateRequest": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/id"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "registerRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "event",
+        "id"
+      ]
+    },
+    "unregisterRequest": {
+      "type": "object",
+      "properties": {
+        "event": {
+          "$ref": "#/definitions/eventType"
+        },
+        "id": {
+          "$ref": "#/definitions/eventId"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "activateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "socket": {
+          "$ref": "#/definitions/socket"
+        }
+      },
+      "required": [
+        "socket"
+      ]
+    },
+    "stateResult": {
+      "type": "object",
+      "default": null,
+      "description": "",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/state"
+        }
+      },
+      "required": [
+        "state"
+      ]
+    },
+    "getListenersResult": {
+      "type": "object",
+      "properties": {
+        "listeners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/listener"
+          }
+        }
+      },
+      "required": [
+        "listeners"
+      ]
+    }
+  },
+  "methods": {
+    "activate": {
+      "summary": "Activate an app session and spawn a socket",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/activateRequest"
+      },
+      "result": {
+        "$ref": "#/definitions/activateResult"
+      },
+      "errors": [
+      ]
+    },
+    "stop": {
+      "summary": "Stops an app session and closes the socket",
+      "events": [
+        "StateChangeEvent"
+      ],
+      "params": {
+        "$ref": "#/definitions/stopRequest"
+      },
+      "result": {
+        "$ref": "#/common/results/void"
+      },
+      "errors": [
+      ]
+    },
+    "deactivate": {
+      "summary": "Suspends an app session",
```

Comment:
"summary": "Suspents an app session (Moves it to 'inactive' state).\n\nCauses server to terminate A/V playback sessions.\nIf App session server state is changed, then StateChangeEvent 'inactive' will be sent.\nIf App session server is already in 'inactive' state, then method call will succeed but no StateChangeEvent will be sent.",